### PR TITLE
nagios: allow to filter stdout and stderr from event comparison

### PR DESCRIPTION
**Pull request by xp-1000**
Hello 

related to https://github.com/signalfx/signalfx-agent/pull/1553 

it seems users want this to be customizable because there are as much scripts with different and same output from nagios world.

thanks for the quick merge of the previous one it will be enough to unstuck my colleague.

### CODE REVIEW

File: pkg/monitors/nagios/nagios.go [hunk 1]
```diff
@@ -38,6 +38,12 @@ type Config struct {
 	Service string `yaml:"service" validate:"required"`
 	// The max execution time allowed in seconds before sending SIGKILL
 	Timeout int `yaml:"timeout" default:"9"`
+	// If `false` and change is detected on `stdout` compared to the last
+	// event it will send a new one
+	FilterStdOut bool `yaml:"filterStdOut" default:"false"`
```
Comment (keitwb) [inline pkg/monitors/nagios/nagios.go:43]: I would call this `IgnoreStdOut`, etc.  "Filter" is not very clear.

File: pkg/monitors/nagios/nagios.go [hunk 2]
```diff
@@ -95,7 +101,10 @@ func (m *Monitor) Configure(conf *Config) error {
 		properties := makeProperties(state, err, stdout, stderr)
 		// Some scripts could produce different output (and stderr) for
 		// each interval "normally", so we do not want to compare them
-		diffProperties := filterProperties(properties)
+		m.logger.Warn(conf.FilterStdOut)
```
Comment (keitwb) [inline pkg/monitors/nagios/nagios.go:104]: Remove these two log lines.

Comment (xp-1000) [reply pkg/monitors/nagios/nagios.go:104]: again, sorry

File: pkg/monitors/nagios/nagios.go (hunk 1 above)
Comment (xp-1000) [reply pkg/monitors/nagios/nagios.go]: done

**Review by keitwb — APPROVED** [review]

**Pull request merged**